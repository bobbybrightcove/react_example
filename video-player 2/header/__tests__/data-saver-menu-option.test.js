import React from 'react';
import renderer from 'react-test-renderer';
import DataSaverMenuOption from '../data-saver-menu-option';

describe('DataSaverMenu', () => {
  it('should render with no selection', () => {
    const onClick = jest.fn();
    const component = renderer.create(
      <DataSaverMenuOption
        onClick={onClick}
        option={{
          label: 'label',
          description: 'description',
          bitrate: 100000,
        }}
      />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
