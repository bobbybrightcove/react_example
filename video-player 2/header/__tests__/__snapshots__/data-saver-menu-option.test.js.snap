// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataSaverMenu should render with no selection 1`] = `
<div
  className="dataSaverOption"
  onClick={[Function]}
>
  <div
    className="dataSaverOptionLeft"
  >
    <div
      className="dataSaverOptionName"
    >
      label
    </div>
    <div
      className="dataSaverOptionDesc"
    >
      description
    </div>
  </div>
  <div
    className="dataSaverOptionRight"
  >
    <div
      className="checkboxWrapper"
    >
      <img
        alt={false}
        src="checkbox-unchecked.svg"
      />
    </div>
  </div>
</div>
`;
